<ui version="4.0" >
 <class>grafico</class>
 <widget class="QWidget" name="grafico" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Gráfico</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>:/images/list2.svg</normaloff>:/images/list2.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableWidget" name="tablaDatos" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <property name="sizeConstraint" >
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QwtPlot" name="GPlot" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="MinimumExpanding" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>503</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="guardarButton" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>&amp;Guardar Imágen</string>
           </property>
           <property name="icon" >
            <iconset resource="libroK.qrc" >
             <normaloff>:/images/librok-37.png</normaloff>:/images/librok-37.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="imprimirButton" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>&amp;Imprimir</string>
           </property>
           <property name="icon" >
            <iconset resource="libroK.qrc" >
             <normaloff>:/graf/Print.png</normaloff>:/graf/Print.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6" />
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="libroK.qrc" />
 </resources>
 <connections/>
</ui>
